Junior-Devs/2020-Junior-Devs-Project
# Storybook jest setup

### Overview of changes:

- Added `jest` + `react-testing-library` to dependencies
- Added `babel` config
- Added` storybook`
- Added a template for component + test + story 

### Test instructions:

- Run the frontend app (`cd frontend` - `npm run dev`)
- Run `npm run storybook`, check that it builds
- Run `npm run test`, see it pass
